2020-01-07 11:55:46,799 INFO [root] [__init__.py(<module>:5)] - TEST日志器能不能正常工作
2020-01-07 11:55:47,502 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': True, 'message': '操作成功！', 'code': 10000, 'data': '72ffd8bd-e818-413d-92c8-fd7cddd916c3'}
2020-01-07 11:55:47,627 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:47,752 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:47,861 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:48,002 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:48,158 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:48,267 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
2020-01-07 11:55:48,377 INFO [root] [hr_test_login_para.py(test_login:27)] - 登录接口返回数据为:{'success': False, 'message': '用户名或密码错误', 'code': 20001, 'data': None}
